app/helpers/application_helper.rb

```diff
@@ -15,19 +15,9 @@ def duration_display(minutes)
     hours = minutes / 60
     minutes = minutes - (hours * 60)
 
-    minute_string = minutes > 0 ? "#{minutes}m" : ""
-    hour_string = hours > 0 ? "#{hours}h" : ""
+    minute_string = minutes > 0 ? "#{minutes}" : ""
+    hour_string = hours > 0 ? "#{hours}" : ""
 
-    "#{hour_string} #{minute_string}".strip
-  end
-
-  def long_duration_display(minutes)
-    hours = minutes / 60
-    minutes = minutes - (hours * 60)
-
-    minute_string = minutes > 0 ? "#{minutes} #{'minute'.pluralize(minutes)}" : ""
-    hour_string = hours > 0 ? "#{hours} #{'hour'.pluralize(hours)}" : ""
-
-    "#{hour_string} #{minute_string}".strip
+    "#{hour_string}:#{minute_string}".strip
```

Comment:
This implementation yield some odd results, based on how the previous one worked. In the previous implementation, I didn't want to have results like "0h 1m" or "1h 0m", so I made it produce "1m" and "1h" instead, respectively. I did this by setting hour_string or minute string to an empty string if the hour or minute was 0, respectively, up on lines 18 and 19.

All that to say, to simplify this a bit, this line should be `"#{'%02d' % hours}:#{'%02d' % minutes}"` and lines 18 and 19 removed.